base/compiler/abstractinterpretation.jl

```diff
@@ -1277,6 +1303,8 @@ function typeinf_local(interp::AbstractInterpreter, frame::InferenceState)
     W = frame.ip
     s = frame.stmt_types
     n = frame.nstmts
+    nargs = frame.nargs
+    isva = let def = frame.linfo.def; isa(def, Method) && def.isva; end
```

Comment:
style nit: the `;` is mostly for convenient REPL usage, but we generally try to avoid packing statements on a line (the let block also isn't essential and just adds line noise in this case)
```suggestion
    def = frame.linfo.def
    isva = isa(def, Method) && def.isva
```